nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -81,6 +81,7 @@
     public static final String PROCESSOR_SCHEDULING_TIMEOUT = "nifi.processor.scheduling.timeout";
     public static final String BACKPRESSURE_COUNT = "nifi.queue.backpressure.count";
     public static final String BACKPRESSURE_SIZE = "nifi.queue.backpressure.size";
+    public static final String COMMANDCONTROL_LISTEN_PORT = "nifi.commandcontrol.listen.port";
```

Comment:
Command & Control already has a different meaning in the NiFi ecosystem context: https://cwiki.apache.org/confluence/display/MINIFI/C2+Design

In the code we already refer to it as a "bootstrap listener", so maybe `nifi.bootstrap.port` and `nifi.bootstrap.listener.port` are better names for the properties?

Reply:
Thanks for taking the time to review this!

Makes sense. I've renamed the property to ``nifi.bootstrap.port``, and rebased to master.